### PR TITLE
Feature/162727514/user create article

### PR DESCRIPTION
**Pull request by rajeman**
#### What does this PR do?
- It ensures an authenticated user can create an article

#### Description of Task to be completed?
- [x] Add create article feature
- [x] Write unit tests

#### How should this be manually tested?
- goto `/create-article` after signing in and enter the required fields

#### What are the relevant pivotal tracker stories?
- [#162727514](https://www.pivotaltracker.com/story/show/162727514)

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-  "presets": ["@babel/preset-env", "@babel/preset-react"]
+  "presets": ["@babel/preset-env", "@babel/preset-react"],
+  "plugins": [
+    "@babel/plugin-proposal-class-properties"
```
Comment (ebenezerdon) [inline .babelrc:4]: screen shot 2019-02-17 at 4 44 54 pm
Do you think adding this plugin to our `webpack config` might help?

Comment (rajeman) [reply .babelrc:4]: I don't get you. Help with what?

Comment (ebenezerdon) [reply .babelrc:4]: screen shot 2019-02-17 at 4 31 09 pm
I initially sent a screenshot of the logs. The app is crashing.

Comment (rajeman) [reply .babelrc:4]: I will look into it

**Review by andela-mabdussalam — APPROVED** [review]

**Pull request merged**